airbyte-integrations/connectors/source-sentry/unit_tests/integration/test_issues_stream.py

```diff
@@ -0,0 +1,162 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+
+import json
+from unittest import TestCase
+
+from airbyte_cdk.test.catalog_builder import CatalogBuilder
+from airbyte_cdk.test.entrypoint_wrapper import read
+from airbyte_cdk.test.mock_http import HttpMocker, HttpRequest, HttpResponse
+from airbyte_cdk.test.state_builder import StateBuilder
+from airbyte_protocol.models import SyncMode
+from config_builder import ConfigBuilder
+from source_sentry.source import SourceSentry
+
+
+class TestEvents(TestCase):
+    def catalog(self, sync_mode: SyncMode = SyncMode.full_refresh):
+        return CatalogBuilder().with_stream(name="issues", sync_mode=sync_mode).build()
+
+    def config(self):
+        return ConfigBuilder().build()
+
+    def state(self):
+        return StateBuilder().with_stream_state(stream_name="issues", state={"lastSeen": "2023-01-01T00:00:00.0Z"}).build()
+
+    @HttpMocker()
+    def test_read(self, http_mocker: HttpMocker):
+        response = [
```

Comment:
nit: please move responses to `unit_tests/resource/http/response`

Reply:
updated